sprotty-vscode-extension/src/sprotty-vscode-extension.ts

```diff
@@ -24,13 +24,16 @@ export abstract class SprottyVscodeExtension {
     protected readonly webviewMap = new Map<string, SprottyWebview>();
     protected singleton?: SprottyWebview;
 
-    constructor(readonly extensionPrefix: string, readonly context: vscode.ExtensionContext) {
+    constructor(readonly extensionPrefix: string, readonly context: vscode.ExtensionContext) {
+        console.log("SprottyVscodeExtension constructor");
```

Comment:
Please remove such log outputs.

Reply:
Sorry, did not notice I left them in here. Removed.